python/ambassador/ir/irhost.py

```diff
@@ -60,52 +60,45 @@ def setup(self, ir: 'IR', aconf: Config) -> bool:
 
                 if tls_ss:
                     # OK, we have a TLS secret! Fire up a TLS context for it, if one doesn't
-                    # already exist, and if the ACME provider isn't set to "none".
+                    # already exist.
 
-                    acme = self.get("acmeProvider") or {}
+                    ctx_name = f"{self.name}-context"
 
-                    acme_authority = acme.get('authority') or 'none'
-
-                    if acme_authority.lower() != "none":
-                        ctx_name = f"{self.name}-context"
-
-                        if ir.has_tls_context(ctx_name):
-                            ir.logger.info(f"Host {self.name}: TLSContext {ctx_name} already exists")
-                        else:
-                            ir.logger.info(f"Host {self.name}: creating TLSContext {ctx_name}")
+                    if ir.has_tls_context(ctx_name):
+                        ir.logger.info(f"Host {self.name}: TLSContext {ctx_name} already exists")
+                    else:
+                        ir.logger.info(f"Host {self.name}: creating TLSContext {ctx_name}")
 
-                            new_ctx = dict(
-                                rkey=self.rkey,
-                                name=ctx_name,
-                                namespace=self.namespace,
-                                location=self.location,
-                                hosts=[ self.hostname or self.name ],
-                                secret=tls_name
-                            )
+                        new_ctx = dict(
+                            rkey=self.rkey,
+                            name=ctx_name,
+                            namespace=self.namespace,
+                            location=self.location,
+                            hosts=[ self.hostname or self.name ],
+                            secret=tls_name
+                        )
 
-                            if not os.environ.get('AMBASSADOR_NO_TLS_REDIRECT', None):
-                                new_ctx['redirect_cleartext_from'] = 8080
+                        # if not os.environ.get('AMBASSADOR_NO_TLS_REDIRECT', None):
+                        #     new_ctx['redirect_cleartext_from'] = 8080
```

Comment:
What was that ever for?  Just as a hack for KAT?

Reply:
Many of the KAT tests did a poor job of handling the automagic redirect on 8080 that used to be there. This was a crutch that I expect to be deleting shortly (I have to do a full test run to make sure it's really no longer used).